src/pixi.inspector.js

```diff
@@ -19,7 +19,7 @@
 	}
 
 	var inspector = {
-		
```

Comment:
My editor automatically removed lots of trailing whitespace, I'll call out significant areas of the diff